src/test/java/ru/mtsbank/fintech/starter_tests/test_config/TestsConfiguration.java

```diff
@@ -6,34 +6,48 @@
 import ru.mts.animals.Bear;
 import ru.mts.animals.Cat;
 import ru.mts.animals.Fish;
+import ru.mts.animals_creators.AnimalFactory;
 import ru.mts.animals_creators.CreateAnimalServiceImpl;
 import ru.mtsbank.fintech.animal_repository.AnimalRepositoryImpl;
 
 import java.math.BigDecimal;
 import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 @org.springframework.boot.test.context.TestConfiguration
 
 public class TestsConfiguration {
```

Comment:
Ради меня и дёргающегося глаза, плизки :)

```suggestion
import org.springframework.boot.test.context.TestConfiguration;

@TestConfiguration
public class TestsConfiguration {
```